ext/reflection/php_reflection.stub.php

```diff
@@ -102,8 +102,8 @@ public function getAttributes(?string $name = null, int $flags = 0): array {}
 
 class ReflectionFunction extends ReflectionFunctionAbstract
 {
-    /** @param string|Closure $name */
-    public function __construct($name) {}
+    /** @param Closure|string $name */
+    public function __construct($function) {}
```

Comment:
Should be
```suggestion
    public function __construct(Closure|string $function) {}
```
now, or? As you're using the proper zpp implementation now. Similarly for some of the other cases below.

Reply:
No :( That's not possible, `gen_func.php` throws an exception with "Not implemented yet". And I haven't checked why I could do about it.

Reply:
Oh you're right. Let me check if I can quickly add this.

Reply:
If you manage to implement the arginfo code, I can finish the `gen_stub` part.

Reply:
I've added something in https://github.com/php/php-src/commit/1f8a93abaafcdef776f342634af167c466cae34c. It might be slightly untested though :P

Reply:
Ohh, that was much-much easier than I imagined ^^ Thanks!

Reply:
Apparently, it worked based on the reflection tests about the reflection classes. ^^